databuilder/main.py

```diff
@@ -161,7 +161,11 @@ def get_sql_strings(query_engine, variable_definitions):
     sql = clause_as_str(results_query, dialect)
     sql_strings.append(f"-- Results query\n{sql}")
 
-    assert not cleanup_queries, "Support these once tests exercise them"
+    for n, query in enumerate(cleanup_queries, start=1):
```

Comment:
The `n` confused me. Surely `i`?

Reply:
I was following the pattern used elsewhere, but I agree with your instinct

Reply:
Oh yeah, that was me, sorry! And the pattern's probably replicated to a few different files now. Worth changing all these loop variables to `i` in one commit I think.

Reply:
done